doc/pcre2api.3

```diff
@@ -1296,14 +1296,15 @@ documentation for more details.
 .sp
   PCRE2_CONFIG_JITTARGET
 .sp
-The \fIwhere\fP argument should point to a buffer that is at least 64 code
-units long. (The exact length required can be found by calling
-\fBpcre2_config()\fP with \fBwhere\fP set to NULL.) The buffer is filled with a
-string that contains the name of the architecture for which the JIT compiler is
-configured, for example "x86 32bit (little endian + unaligned)". If JIT support
-is not available, PCRE2_ERROR_BADOPTION is returned, otherwise the number of
-code units used is returned. This is the length of the string, plus one unit
-for the terminating zero.
+The \fIwhere\fP argument should point to a buffer that is suitably aligned and
+wide enough to hold the full string. The exact length required is returned when
+calling \fBpcre2_config()\fP with \fBwhere\fP set to NULL first. The buffer is
+filled with a string that contains the name of the architecture for which the JIT
+compiler is configured at build time, for example, a 64-bit ARM CPU that supports
+the Armv8.1 extension writes "ARM-64 (LSE) 64bit (little endian + unaligned)".
+If JIT support is not available, PCRE2_ERROR_BADOPTION is returned, otherwise
+the number of code units used is returned. This is the length of the string,
+plus one code unit for the NUL character.
```

Comment:
this is missing an historical overview and an explanation of why the use of static buffers is discouraged.